tutorials/llama_index_search_and_retrieval_tutorial.ipynb

```diff
@@ -0,0 +1,819 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    print("Embedding Dimension")
</pre><p>do we need this print?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/891/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>I'll show the embedding itself. I want to call attention to the fact that the embeddings are available in addition to the query and response.</p>

Reply:
<p>Easy to miss this detail.</p>